enterprise_access/settings/base.py

```diff
@@ -466,7 +466,8 @@ def root(*path_fragments):
 BRAZE_APPROVE_NOTIFICATION_CAMPAIGN = ''
 BRAZE_DECLINE_NOTIFICATION_CAMPAIGN = ''
 BRAZE_AUTO_DECLINE_NOTIFICATION_CAMPAIGN = ''
-BRAZE_ASSIGNMENT_REMINDER_NOTIFICATION_CAMPAIGN = ''
+BRAZE_ASSIGNMENT_REMINDER_LOGISTRATE_CAMPAIGN = 'a5fbad5c-73b2-4aa0-a5be-03045f1043ce'
+BRAZE_ASSIGNMENT_REMINDER_NOTIFICATION_CAMPAIGN = '2009d64a-9980-4fb6-a6a3-005b9afed416'
```

Comment:
Are these the stage/prod specific campaign identifiers? If so, they should not be committed in the repo, similar to the others that are default to `''`. The UUIDs will come from our 2U/edX specific config.